libs/design/src/lib/iconized/iconized.tsx

```diff
@@ -1,25 +1,27 @@
 import styles from './iconized.module.scss';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { IconDefinition } from '@fortawesome/fontawesome-svg-core';
+import { IconDefinition, SizeProp } from '@fortawesome/fontawesome-svg-core';
 import { HTMLAttributes } from 'react';
 import classnames from 'classnames';
 
 export interface IconizedProps extends HTMLAttributes<HTMLDivElement> {
   icon: IconDefinition;
+  size?: SizeProp;
 }
 
 export function Iconized({
   icon,
   children,
   className,
+  size,
   ...passThrough
 }: IconizedProps) {
   return (
     <div
       className={classnames(styles['container'], className)}
       {...passThrough}
     >
-      <FontAwesomeIcon icon={icon} />
+      <FontAwesomeIcon icon={icon} size={size} />
```

Comment:
I wonder if we shouldn't have a `icon: FontAwesomeIconProps` instead of adding a prop each time we need to pass to Icon ...
May be overkilling for our behavior but it demonstrate we can use lib's types (typescript) and spread props (just like we did with `extends HTMLAttributes<HTMLDivElement>` )

Reply:
Good point actually